Jenkinsfile

```diff
@@ -11,7 +11,7 @@ pipeline {
   }
   triggers {
     pollSCM('H/5 * * * *')
-    cron('@daily')
+    cron('H H * * *')
```

Comment:
hah, so this would be daily, which is what I had assumed you wanted.. I believe hourly would be H * * *

Reply:
Sigh; I need to sleep (really).  Fixing.

The good news is that I can verify the two values were both set in the UI, automagically!

<img width="1792" alt="screen shot 2018-02-13 at 7 34 40 am" src="https://user-images.githubusercontent.com/387249/36158324-ec0e2960-1090-11e8-843c-49571f26d77e.png">